src/components/QuranReader/index.tsx

```diff
@@ -51,12 +55,12 @@ const QuranReader = ({
 }: QuranReaderProps) => {
   const isVerseData = quranReaderDataType === QuranReaderDataType.Verse;
   const isTafsirData = quranReaderDataType === QuranReaderDataType.Tafsir;
-  const isSideBarVisible = useSelector(selectNotes).isVisible;
-  const quranReaderStyles = useSelector(selectQuranReaderStyles) as QuranReaderStyles;
-  const { selectedTranslations, isUsingDefaultTranslations } = useSelector(
-    selectTranslations,
-  ) as TranslationsSettings;
-  const { selectedTafsirs, isUsingDefaultTafsirs } = useSelector(selectTafsirs) as TafsirsSettings;
+  const isSideBarVisible = useSelector(selectNotes, shallowEqual).isVisible;
+  const quranReaderStyles = useSelector(selectQuranReaderStyles, shallowEqual) as QuranReaderStyles;
+  const selectedTranslations = useSelector(selectSelectedTranslations, areArraysEquals);
+  const isUsingDefaultTranslations = useSelector(selectIsUsingDefaultTranslations);
```

Comment:
is it useful to use `shallowEqual` here?

Reply:
yeah, I refactored it from shallowEqual into multiple selector actually. Because `selectedTranslations` is an array. so `shallow` will not work. 

Similar to your question related to `showTooltipFor`. it won't work with array inside object